### ISSUE TITLE
A lot of highlight messages

### ISSUE THREAD

**Issue by ndr-brt** [tidalcycles/tidal-listener#20]
For a simple pattern, e.g. `s "bd"`, I receive something like 36/37 messages per cycle, it's expected?

**Comment by yaxu** [comment]
Hmm well that's not completely surprising to me, I think it might run at 20Hz? But I didn't really think this through.. Do the messages make sense so far?

**Comment by ndr-brt** [comment]
Yep, the messages make sense, but are all equals, so every cycle there are 36-37 identical messages, that's not a big problem, but they need to be de-duplicated to permit a seamless highlight effect
